prettyprinter-ansi-terminal/bench/LargeOutput.hs

```diff
@@ -0,0 +1,157 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Is this necessary?

Reply:
Unfortunately we still need `instance Arbitrary Text`